parquet-hadoop/src/main/java/org/apache/parquet/ParquetReadOptions.java

```diff
@@ -42,13 +42,15 @@
   private static final boolean COLUMN_INDEX_FILTERING_ENABLED_DEFAULT = true;
   private static final int ALLOCATION_SIZE_DEFAULT = 8388608; // 8MB
   private static final boolean PAGE_VERIFY_CHECKSUM_ENABLED_DEFAULT = false;
+  private static final boolean BLOOM_FILTER_ENABLED_DEFAULT = false;
```

Comment:
Why don't we enable it by default? I think, if a bloom filter is written, we should use it. In addition, in `HadoopReadOptions` it is enabled by default.

Reply:
Thanks for pointing this out, I will go through all these configurations and clean them up.